src/jit/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Why was this removed?

Reply:
This is done at the root CMakeLists.txt now.